tests/wpt/web-platform-tests/html/semantics/scripting-1/the-script-element/execution-timing/029.html

```diff
@@ -10,6 +10,9 @@
         <div id="log">FAILED (This TC requires JavaScript enabled)</div>
         <p><a href="javascript:log('JS URL')"></a></p>
         <script>log('inline script #1');
+                window.addEventListener("beforeunload", function( event ) {
+                        log('beforeunload event');
```

Comment:
Ditto.

Reply:
This is different from console.log.

Reply:
Oops, didn't look very carefully at that!